public/js/main.js

```diff
@@ -108,41 +108,39 @@ const composeTextarea = elem({
  tagName: 'textarea',
 })
 
-// --- Resize Observer for SCRIPT_CHANNEL textarea height persistence ---
-let debounceTimeout
-const scriptTextareaResizeObserver =
- new ResizeObserver((entries) => {
-  for (let entry of entries) {
-   // Only act if the observed element is the composeTextarea
-   if (entry.target === composeTextarea) {
-    // Debounce the saving operation
-    clearTimeout(debounceTimeout)
-    debounceTimeout = setTimeout(() => {
-     // Check if we are currently on the SCRIPT_CHANNEL
-     if (
-      getUrlData().channel === SCRIPT_CHANNEL
-     ) {
-      const height =
-       composeTextarea.offsetHeight // Get the actual rendered height
-      if (height > 0) {
-       // Only save valid heights
-       localStorage.setItem(
-        '𝓢.height',
-        `${height}px`
-       )
-       // console.log(`Saved S channel height: ${height}px`);
-      }
-     }
-    }, 300) // Debounce duration (300ms)
-   }
-  }
- })
-
-// Start observing the textarea
-scriptTextareaResizeObserver.observe(
- composeTextarea
-)
-// -------------------------------------------------------------------
```

Comment:
We need to undelete this part of the code, you can copy this block from https://github.com/tagmein/tagme.in/blob/main/public/js/main.js#L111